src/auth/auth.controller.ts

```
@@ -0,0 +1,125 @@
+
+  // Google auth initiation endpoint
+  @Get('google')
+  @UseGuards(AuthGuard('google'))
+  googleAuth() {
+    // This is handled by Passport
+    // The initiation doesn't need any code in the method body
+  }
+
+  // Google callback endpoint
+  @Get('google/callback')
+@UseGuards(AuthGuard('google'))
```

ESLint

[error] Insert `··`

(prettier/prettier)